invoice2data/template.py

```diff
@@ -212,6 +212,9 @@ def extract_lines(self, content, output):
                 'last_line' not in self['lines']:
             self['lines']['first_line'] = self['lines']['line']
         for line in re.split(self.options['line_separator'], content):
+            # if the line has empty lines in it , skip them
+            if  re.search(r'''^\s*\n$''', line) or not line:
```

Comment:
no need for the big guns: https://docs.python.org/2/library/string.html#string.strip